README.md

```diff
@@ -47,9 +47,12 @@ $ qemu-system-i386 pinpog
 
 ### Windows
 
-[TBD](https://github.com/tsoding/pinpog/issues/65)
+1. Download UltraISO https://www.ultraiso.com/
```

Comment:
Should be a clickable link. https://daringfireball.net/projects/markdown/syntax#link